src/org/loklak/api/search/QuoraProfileScraper.java

```diff
@@ -307,7 +307,7 @@ private Timeline2 scrapeQues(BufferedReader br, String url) {
         return quesList;
     }
 
-    private Timeline2 scrapeProfile(BufferedReader br, String url) {
+    public Timeline2 scrapeProfile(BufferedReader br, String url) {
```

Comment:
there is a method in TwitterScraperTest.java for accessing private members. You may simply implement it for these tests